app/models/Post.py

```diff
@@ -12,14 +12,15 @@ class Post(db.Model):
     Represnts a post (e.g. challenge)
     """
 
-    id = db.Column(db.Integer, primary_key=True, unique=True, autoincrement=True)
+    __table_args__ = {'extend_existing': True}
```

Comment:
This is out of date wrt master. Please merge changes from master into this branch.